blocks/medium-promo-block/README.md

```diff
@@ -1,52 +1,51 @@
 # `@wpmedia/medium-promo-block`
 
-_Block containing a medium promo component. Please provide a 1-2 sentence description of what the block is and what it does._
+Medium Manual Promo Block provides a way for a PageBuilder user to create a data-driven promo item using content data.
```

Comment:
```suggestion
Medium Promo Block provides a way for a PageBuilder user to create a data-driven promo item using content data.
```

this is not the manual promo

Reply:
@ecarlisle it looks like this has not been addressed. When you set "ready for review", our team expectation is that all reviews have either been addressed or commented on

Reply:
TYVM @JackHowa ! Corrected.

Reply:
updated